stable/grafana/values.yaml

```diff
@@ -154,6 +153,35 @@ persistence:
   # subPath: ""
   # existingClaim:
 
+initChownData:
+  ## If false, data ownership will not be reset at startup
+  ## This allows the prometheus-server to be run with an arbitrary user
+  ##
+  enabled: true
+
+  ## initChownData container name
+  ##
+  name: init-chown-data
```

Comment:
lets keep the name hard coded in the template. it is not likely that someone will want to change that ;)

Reply:
I agree with that.